app/src/main/java/com/example/parkingsystemjava/mvp/presenter/DialogFragmentPresenter.java

```diff
@@ -0,0 +1,29 @@
+package com.example.parkingsystemjava.mvp.presenter;
+
+import com.example.parkingsystemjava.mvp.contract.DialogFragmentContract;
+import com.example.parkingsystemjava.mvp.listener.CustomListener;
+
+public class DialogFragmentPresenter implements DialogFragmentContract.PresenterContract {
+    private DialogFragmentContract.ViewContract view;
+    private CustomListener customListenerDialogFragment;
+
+    public DialogFragmentPresenter(DialogFragmentContract.ViewContract view, CustomListener customListenerDialogFragment) {
+        this.view = view;
+        this.customListenerDialogFragment = customListenerDialogFragment;
+    }
+
+    @Override
+    public void onSaveButtonPressed() {
+        String spaces = view.getParkingsAvailable();
+        if (!spaces.equals("")) {
```

Comment:
pls avoid hardcodes

Reply:
Yes, try to use a native method to check if it is empty

Reply:
done, changed for "!spaces.isEmpty()" suggested by SartiMau